src/components/form/form.stories.mdx

```diff
@@ -77,7 +77,31 @@ When either `errorCount` or `warningCount` or both are provided, summary with th
 </Canvas>
 
 <Canvas>
-  <Story name="with both errors and warnings summary" story={stories.WithBothErrorsAndWarningsSummary} />
+  <Story
+    name="with both errors and warnings summary"
+    story={stories.WithBothErrorsAndWarningsSummary}
+  />
+</Canvas>
+
+### Optional and Required Fields
+
+When `isOptional` prop is passed.
+
+<Canvas>
+  <Story name="with optional" story={stories.WithOptionalLabel} />
+</Canvas>
+
+When `required` prop is passed.
+
+<Canvas>
+  <Story name="required" story={stories.WithRequiredLabel} />
+</Canvas>
```

Comment:
**suggestion**: I think just having the story with both the optional and required fields should be sufficient here:


```suggestion
```